enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -0,0 +1,165 @@
+---
+title: windows-machine-config-operator
+authors:
+  - "@ravisantoshgudimetla"
+  - "@aravindhp"
+reviewers:
+  - "@sdodson"
+  - "@derekwaynecarr"
+  - "@openshift-team-cloud"
+  - "@openshift-team-mco"
+approvers:
+  - "@sdodson"
+  - "@derekwaynecarr"
+creation-date: 2020-05-28
+last-updated: 2020-05-28
+status: implementable
+replaces:
+  - "/enhancements/windows-containers/ansible-dev-preview.md"
+---
+
+# Windows Machine Config Operator
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to allow a cluster administrator to add a
+Windows compute node with a prescribed configuration to an installer provisioned
+OpenShift 4.6 cluster on AWS, Azure and vSphere,configured with Hybrid OVN
+networking, as a day 2 operation and enable scheduling of Windows workloads.
+
+## Motivation
+
+The main motivation behind this enhancement is to satisfy customer
+requirement of being able to run Windows workloads on OpenShift clusters.
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Provide workflows for adding Windows instances provisioned using the Machine
+  API to OpenShift clusters.
+* Perform all the required steps within the node for it to be an OpenShift
```

Comment:
should we add a more specific word here for Openshift compute node since we do the steps so the node can join the cluster as a worker node with some configurations.